build.sh

```diff
@@ -40,11 +40,13 @@ configure_yum_repos() {
     # For Fedora we'll add a FAHC and COPR repo
     if [ -n "${ISFEDORA}" ]; then
         # Enable FAHC https://pagure.io/fedora-atomic-host-continuous
-        # so we have ostree/rpm-ostree git master for our :latest
+        # but as disabled.  Today FAHC isn't multi-arch.  But let's
+        # add it so that anyone on x86_64 who wants to test the latest
+        # ostree/rpm-ostree can easily do so.
         # NOTE: The canonical copy of this code lives in rpm-ostree's CI:
         # https://github.com/projectatomic/rpm-ostree/blob/d2b0e42bfce972406ac69f8e2136c98f22b85fb2/ci/build.sh#L13
         # Please edit there first
-        echo -e '[fahc]\nmetadata_expire=1m\nbaseurl=https://ci.centos.org/artifacts/sig-atomic/fahc/rdgo/build/\ngpgcheck=0\n' > /etc/yum.repos.d/fahc.repo
+        echo -e '[fahc]\nenabled=0\nmetadata_expire=1m\nbaseurl=https://ci.centos.org/artifacts/sig-atomic/fahc/rdgo/build/\ngpgcheck=0\n' > /etc/yum.repos.d/fahc.repo
```

Comment:
I think we need to drop the `exclude=` higher up too, right?

Actually, let's just nuke this whole function? We don't need the Ignition COPR anymore either right? (Looks like it doesn't have aarch64 builds either anyway).

Reply:
Yeah, I should have noted I didn't test this :angel:

Reply:
+1 for removing fahc and copr